src/language-css/printer-postcss.js

```diff
@@ -249,15 +249,9 @@ function genericPrint(path, options, print) {
       );
     }
     case "media-unknown": {
-      return adjustStrings(node.value, options);
+      return node.value;
```

Comment:
We don't should change nothing in `unknown` nodes, because it is potential can break output. safe output as is